dandiapi/api/admin.py

```diff
@@ -157,6 +159,28 @@ class ZarrArchiveAdmin(admin.ModelAdmin):
     list_display = ['id', 'zarr_id', 'name']
     list_display_links = ['id', 'zarr_id', 'name']
 
+    @admin.action(description='Ingest selected zarr archives')
+    def ingest_zarr_archive(self, request, queryset):
+        for zarr in queryset:
+            zarr: ZarrArchive
+            ingest_zarr_archive.delay(zarr_id=(str(zarr.id)))
+
+        # Return message
+        self.message_user(
```

Comment:
if this is per zarr archive, the dandiset i'm uploading will have 2600. it would be nice not to get that many messages :)

Reply:
It only sends one message here. So in the admin panel, if you selected 2600 zarr archives to manually ingest, there would just be one green message at the top that reads

`2600 zarr archives have begun ingesting.`

Reply:
thank you